Xamarin.Essentials/Preferences/Preferences.ios.tvos.watchos.cs

```diff
@@ -2,6 +2,7 @@
 using System.Globalization;
 using Foundation;
 
+#nullable disable
 namespace Xamarin.Essentials
 {
     public static partial class Preferences
```

Comment:
I have no clue how to correctly differentiate between Nullable<ValueType> and syntax sugar, so I just disabled it until I know how to handle it

Reply:
Okay, after some further investigation and thought I have come to a conclusion. There are two possible ways of handling this. Either constrain two methods to either struct or class and use them directly, e.g. PlatformGetValueType<T> and PlatformGetString(), so that we can annotate and use nullable directly, or use the null forgiving operator to lie to the compiler that a value passed is not null even though it may be, thus keeping the method signature intact. This would also mean lying when returning a value, since null may be returned. This second approach could be contained to the internals so that our public interface is null compliant, internally this would feel a little bit hacky though